saplings/circuits/src/components/NodeCard.scss

```diff
@@ -0,0 +1,125 @@
+/**
+ * Copyright 2018-2020 Cargill Incorporated
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.node-card {
+  padding: 1rem;
+  width: 100%;
+
+  .not-visible {
```

Comment:
This style might not be necessary (see prev. comment) but usually you want to use `display: none` rather than `visibility: hidden`. With `visibility: hidden` the element still takes up space but is just invisible which can cause issues (there was a bug in Gameroom due to that). So I'd recommend only using the visibility property if you're sure thats what you want.

Reply:
In this case, visibility: hidden makes sense. Otherwise the checkmark to select a node will not be in the same place for local and not local nodes.